pkg/registry/core/pod/rest/log.go

```diff
@@ -90,6 +91,11 @@ func (r *LogREST) Get(ctx context.Context, name string, opts runtime.Object) (ru
 
 	if !utilfeature.DefaultFeatureGate.Enabled(features.PodLogsQuerySplitStreams) {
```

Comment:
Do we prefer this or just to "fix" validation to not assert Required when nil?  We already have this:

pkg/kubelet/server/server.go:
```
 727     if utilfeature.DefaultFeatureGate.Enabled(features.PodLogsQuerySplitStreams) {                                              
 728         // Even with defaulters, logOptions.Stream can be nil if no arguments are provided at all.
 729         if logOptions.Stream == nil {
 730             // Default to "All" to maintain backward compatibility.
 731             logOptions.Stream = ptr.To(v1.LogStreamAll)
 732         } 
 733     }
```

I think that's a little bit cleaner?

Reply:
Yea, I like this. Went ahead and added this.